season/conditionals.md

```diff
@@ -8,15 +8,20 @@ Chartreuse is my favorite color. Magenta is my second favorite. Periwinkle is my
 ```ruby
 color = "Magenta"
 
-# write your code here
+if color == "Magenta"
+	puts "#{color} is my favorite color."
+end 
 ```
 
```

Comment:
Double check your Sublime settings to make sure that you're indenting with spaces and a tab width of 2 - otherwise you get funky tabbing in GitHub^^